app/frontend/src/components/Answer/AnswerStreaming.tsx

```diff
@@ -0,0 +1,67 @@
+import { useMemo } from "react";
+import { Stack, IconButton } from "@fluentui/react";
+import DOMPurify from "dompurify";
+
+import styles from "./Answer.module.css";
+
+import { AskResponse, getCitationFilePath } from "../../api";
+import { parseAnswerToHtml } from "./AnswerParser";
+import { AnswerIcon } from "./AnswerIcon";
+
+interface Props {
+    streamedAnswer: AskResponse;
+    isSelected?: boolean;
+    onCitationClicked: (filePath: string) => void;
+}
+
+export const AnswerStreaming = ({ streamedAnswer, onCitationClicked }: Props) => {
+    const parsedAnswer = useMemo(() => parseAnswerToHtml(streamedAnswer.answer, onCitationClicked), [streamedAnswer]);
+
+    const sanitizedAnswerHtml = DOMPurify.sanitize(parsedAnswer.answerHtml);
+
+    return (
+        <Stack className={`${styles.answerContainer}`} verticalAlign="space-between">
+            <Stack.Item>
+                <Stack horizontal horizontalAlign="space-between">
+                    <AnswerIcon />
+                    <div>
+                        <IconButton
```

Comment:
Feels like these buttons could just be an IconButtonToolbar component that could be enabled or disabled? That'd be less code repetition.